io.github.agam778.MS-Office-Electron.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.github.agam778.MS-Office-Electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: start-ms-office-electron
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    npm_config_nodedir: /run/build/ms-office-electron/flatpak-node/cache/node-gyp/18.2.4/
```

Comment:
I've tried, but it doesn't work without that.
I will retry removing that once again then

Reply:
Let's see what error it gives without it. Setting `npm_config_nodedir` is a bad idea for 2 reasons: 1. you'll need to manually update it on every electron update and 2. it'll break in case of native node modules needed at build time (e.g. sassc).

Reply:
Ok so basically
- Without the "append-path" thing, yarn command doesn't exist
- Without the "npm_config_nodedir", the dependencies aren't built

Reply:
I see. Try removing `npm_config_nodedir` together with [this](https://github.com/flathub/flathub/pull/3166/files#r912224762) suggestion.

Reply:
But I can't remove the dist command because thats used for building the app?

Reply:
Right, I didn't suggest to remove the dist command.

Reply:
Hmm?

![image](https://user-images.githubusercontent.com/68941022/176998282-7f01b69e-44b5-42d2-87ba-5365dd96b7f1.png)

Reply:
`yarn --cwd main run dist --linux dir` is on the next line.

Reply:
Oh nevermind, that works now.